infrastructure/configs/config.go

```diff
@@ -0,0 +1,68 @@
+package configs
+
+import (
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+// Phase is myply server phase type
+type Phase int64
+
+// Config is myply' configuration instance, singleton
+type Config struct {
+	Phase       Phase
+	MongoURI    string
+	MongoDBName string
+}
+
+const (
+	// Test phase
+	Test Phase = iota
```

Comment:
Thanks, I hadn't thought of that. I'll apply it

Reply:
As you know, it is better to handle the zero value because it is designed much more efficiently to use the variable of the stack than the pointer.

Reply:
Could you please explain a little more? I understand the difference between stack and pointer(heap?) but

I am confused about
> it is designed much more efficiently to use the variable of the stack than the pointer. Therefore, it is better to write to handle the zero value

That means it should be initialized to 1 because there is some mechanism that is internally stored in the heap when I set variable to 0?

I applied zero value handling purely because I want to avoid that it would be used incorrectly in an `if condition` (just like Python treats 0 as false).

Reply:
> That means it should be initialized to 1 because there is some mechanism that is internally stored in the heap when I set variable to 0?

No, it is not stored in the heap unless it is a pointer.

> I applied zero value handling purely because I want to avoid that it would be used incorrectly in an if condition (just like Python treats 0 as false).

Why do you think it's a bad use case?